src/components/home-hero.tsx

```
@@ -303,11 +343,26 @@ export function HomeHero({
-      .map((r) => bySlug.get(r.slug))
-      .filter((i): i is IconEntry => Boolean(i))
+      .map((r) => {
+        const entry = bySlug.get(r.slug);
+        return entry ? { entry, ts: r.ts } : null;
+      })
+      .filter((v): v is { entry: IconEntry; ts: number } => Boolean(v))
       .slice(0, 8);
   }, [recentViewed, icons]);
 
+  function shortAgo(ts: number): string {
+    const s = Math.max(0, Math.floor((Date.now() - ts) / 1000));
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/src/components/home-hero.tsx:355:39
  353 |
  354 |   function shortAgo(ts: number): string {
> 355 |     const s = Math.max(0, Math.floor((Date.now() - ts) / 1000));
      |                                       ^^^^^^^^^^ Cannot call impure function
  356 |     if (s < 60) return "now";
  357 |     const m = Math.floor(s / 60);
  358 |     if (m < 60) return `${m}m`;

(react-hooks/purity)